src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php

```diff
@@ -102,14 +108,18 @@ private function getIsGrantedString(IsGranted $isGranted): string
     {
         $processValue = fn ($value) => \sprintf($value instanceof Expression ? 'new Expression("%s")' : '"%s"', $value);
 
-        $argsString = $processValue($isGranted->attribute);
+        $argsString = $isGranted->callable ? 'callable' : $processValue($isGranted->attribute);
```

Comment:
this should be handled instead `processValue` instead (especially once you don't introduce a new property for the case where we want to vote on a closure as attribute)